apps/glusterfs/dbcommon.go

```diff
@@ -141,6 +143,13 @@ func UpgradeDB(tx *bolt.Tx) error {
 		return err
 	}
 
+	err = fixBlockHostingReservedSize(tx)
```

Comment:
I think we missed it in previous patches but would like to have updates with respect a particular entry within the function. Could we move this under VolumeEntryUpgrade() please.

Reply:
@phlogistonjohn and I discussed about this. It is actually better this way. We should not add any upgrade functions in the middle. Some of the later upgrade functions might have a prior upgrade function as a prerequisite. Keep it this way.